examples/socket.io/package.json

```
@@ -6,7 +6,7 @@
   "sideEffects": false,
   "scripts": {
     "build": "remix build",
-    "dev": "remix dev",
+    "dev": "remix watch",
```

This looks confusing.
Can we just keep it as is, like all other examples please?

```suggestion
    "dev": "remix dev",
```

---

This is motivated by @sergiodxa's comment: https://github.com/remix-run/remix/issues/2860#issuecomment-1102130992